index.html

```diff
@@ -51,35 +61,44 @@ <h1>I’m Sheldon<br>
       <!-- work section -->
          <section class="work-section">
             <div class="card-section">
-                <img src="images/first.png" class="profile">
-                <h2 class="ton">Tonic</h2>
-                <ul class="canopy">
-                    <li class="cano">CANOPY</li>
-                    <li><img src="images/Counter.png"></li>
-                    <li>Back End Dev</li>
-                    <li><img src="images/Counter.png"></li>
-                    <li>2015</li>
-                    
-                </ul>
-
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-                <ul class="tech">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>JavaScript</li>
-                </ul>
-                <a href="#">See Project</a>
 
+                <img src="images/first.png" class="profile">
+                <img src="images/portly.png" class="image-desk">
+
+                <div class="ton-section">
+                    <h2 class="ton">Tonic</h2>
+                    <ul class="canopy">
+                        <li class="cano">CANOPY</li>
+                        <li><img src="images/Counter.png"></li>
+                        <li>Back End Dev</li>
+                        <li><img src="images/Counter.png"></li>
+                        <li>2015</li>
+                        
+                    </ul>
+    
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    
+                    <ul class="tech">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#">See Project</a>
+                </div>
+               
+        
             </div>
 
          </section>
 
          <!-- second section -->
 
          <section class="work-section">
-            <div class="card-section">
+            <div class="card-section" id="dis">
```

Comment:
- [ Required ] I think it would be better if the `works` section is responsive across the following screen sizes, `768px` and `1024px`. I encountered the following error at `1024px`. I would like to suggest that you use `display: grid` in order to divide the projects section into two columns which would make the page responsive.

```HTML
  <section class="work-section">
     <div class="card-section">
       <img src="images/first.png" class="profile">
       <img src="images/portly.png" class="image-desk">
    </div>            
   <div class="ton-section">
     <h2 class="ton">Tonic</h2>
       <ul class="canopy">
         <li class="cano">CANOPY</li>
         <li><img src="images/Counter.png"></li>
         <li>Back End Dev</li>
         <li><img src="images/Counter.png"></li>
        <li>2015</li>                
    </ul>
    
     <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
     <ul class="tech">
         <li>HTML</li>
         <li>CSS</li>
         <li>JavaScript</li>
      </ul>
     <a href="#">See Project</a>
   </div>
</section>
```
```CSS
.work-section {
    display: grid;
    grid-template-columns: 50% 50%;
    flex-direction: column;
    margin: 8.52rem;
    border-radius: 10px;
    border: solid 1px #d6d8db;
    background-color: #fff;
}

.card-section img {
   width: 100%;
}
```

## Your design
![Screenshot from 2023-05-05 12-02-45](https://user-images.githubusercontent.com/16392046/236418673-db1c841e-96f5-481c-8196-156b8ebc478d.png)

## Figma design
![Screenshot from 2023-05-05 12-10-08](https://user-images.githubusercontent.com/16392046/236419299-85988747-1696-40fe-92dc-de2cf648d133.png)